bundles/org.openhab.binding.nikobus/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="nikobus"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+
+    <bridge-type id="pc-link">
+        <label>PC-Link</label>
+        <description>PC-Link via serial connection</description>
+        
+        <config-description>
+            <parameter name="port" type="text" required="true">
+                <label>Port</label>
+                <context>serial-port</context>
+                <description>The serial port used to connect to the Nikobus PC Link.</description>
+            </parameter>
+            <parameter name="refreshInterval" type="integer" max="65535" min="10" required="false">
+                <default>60</default>
+                <label>Refresh interval</label>
+                <description>Refresh interval in seconds.</description>
+            </parameter>
+        </config-description>
+    </bridge-type>
+
+	<thing-type id="push-button">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="pc-link-ip" />
```

Comment:
These `bridge-type-ref` need to be updated to reflect the new bridge.

Reply:
Missed that, thx - I guess such things happen when committing code at 3:00AM :)